cmd/openshift-install/create.go

```diff
@@ -435,42 +436,53 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 		return newAPIError(err)
 	}
 
-	if err := waitForBootstrapConfigMap(ctx, client); err != nil {
-		return err
-	}
+	var platformName string
 
-	if err := waitForStableSNOBootstrap(ctx, config); err != nil {
-		return newBootstrapError(err)
+	if assetStore, err := assetstore.NewStore(command.RootOpts.Dir); err == nil {
+		if installConfig, err := assetStore.Load(&installconfig.InstallConfig{}); err == nil && installConfig != nil {
+			platformName = installConfig.(*installconfig.InstallConfig).Config.Platform.Name()
+		}
```

Comment:
What if in case of error or nil installConfig?

Reply:
Code didn't change, it just moved around in the diff.

Reply:
Not sure that even before it was correct to ignore the errors

Reply:
Yep, but let's not add everything to this PR. It should probably be reported as a bug, fixed and backported.

Reply:
I did not suggest to address this very specific point in the PR, but just noticing the behavior. It does not seem super-critical, since in the rare case where the wait-for command was executed from an assets folder without the install-config the timeout will remain at 30 mins.

Reply:
If we don't have any data to say whether the platform is baremetal or not, we assume it's not.
TBH that doesn't seem like a bug to me.